api/cpp/lib.rs

```diff
@@ -232,6 +232,17 @@ pub extern "C" fn slint_detect_operating_system() -> OperatingSystemType {
 }
 
 #[unsafe(no_mangle)]
-pub extern "C" fn open_url(url: &SharedString) {
+pub extern "C" fn slint_open_url(url: &SharedString) {
     i_slint_core::open_url(url)
 }
+
+#[unsafe(no_mangle)]
+pub extern "C" fn slint_escape_markdown(text: &mut SharedString) -> &SharedString {
+    *text = i_slint_core::escape_markdown(&text).into();
+    text
+}
+
+#[unsafe(no_mangle)]
+pub extern "C" fn slint_parse_markdown(text: &SharedString) -> i_slint_core::api::StyledText {
```

Comment:
StyledText will have a destructor so we can't return it.
It should be `slint_parse_markdown(text: &SharedString, out: &mut StyledText)` or `unsafe fn slint_parse_markdown(text: &SharedString, out: *mut StyledText)` and we use core::ptr::write into it.